posthog/batch_exports/service.py

```diff
@@ -687,7 +687,7 @@ def sync_batch_export(batch_export: BatchExport, created: bool):
             start_at=batch_export.start_at,
             end_at=batch_export.end_at,
             intervals=[ScheduleIntervalSpec(every=batch_export.interval_time_delta)],
-            jitter=(batch_export.interval_time_delta / 6),
+            jitter=min(dt.timedelta(minutes=1), (batch_export.interval_time_delta / 6)),
```

Comment:
Only affects 5 minute batch exports, as hourly and daily batch exports already computed to larger jitter values than 1 minute.